src/events/attributes.rs

```diff
@@ -355,7 +444,7 @@ impl<'a> Iterator for Attributes<'a> {
             ($key:expr, $val:expr) => {
                 Some(Ok(Attribute {
                     key: &self.bytes[$key],
-                    value: Cow::Borrowed(&self.bytes[$val]),
+                    value: normalize_attribute_value(Cow::Borrowed(&self.bytes[$val])),
```

Comment:
I think it is better to provide an API method on `Attribute` to get normalized value only when its needed

Reply:
The spec-compliant value needs to be what you get by default, but I would be OK with a `raw_value()`.

Reply:
But if you just want to iterate over some attributes to skip them (for example, you're search `xmlns` attributes), you'll do unnecessary work.